tests/README.md

```diff
@@ -0,0 +1,45 @@
+# Relay e2e Test Suite
+
+---
+
+## How to run
+
+### 1. Install Docker
+
+```
+https://docs.docker.com/get-docker/
+```
+
+### 2. Build Docker container using official image from playwright
+
+```
+docker run -v $PWD:/tests -w /tests --rm --ipc=host -it mcr.microsoft.com/playwright:v1.23.0-focal /bin/bash
+```
+
+### 3. Install Node Dependencies
+
+```
+npm install
+```
+
+### 4. Install Playwright test dependencies
+
+```
+npx playwright install
+```
+
+### 5. Run Tests
+
+```
+create/update a .env file with the following:
+
+TEST_BASE_URL={/* base will be stage url */}
+TEST_ACCOUNT_FREE=
+TEST_ACCOUNT_PASSWORD=
+```
+
```

Comment:
Actually, since you have all the restmail infra set up already, wouldn't it make more sense to just create a fresh account at the start of the test? Would be nice if it's actually possible to run the tests without needing to set more environment variables, and also makes it more resilient (because if e.g. a test fails to delete all the masks it generated, that won't affect future tests), and makes it possible for the tests to run in parallel (e.g. two developers running the suite at the same time).

Reply:
existing masks shouldn't be an issue for any of the tests as a clean up is done as a before hook in every tests. I've went through the process of creating new accounts at the beginning of tests however it increased test run time significantly. I'm looking into the api account creation that would be part of the next steps for the framework.

Reply:
Ah OK, I thought you were already creating new accounts in the tests, but must've misread :)

And yeah, it's cool that the tests still succeed, but if you inadvertently input your personal account, you might still unwittingly delete all your existing masks :sweat_smile: